<?xml version='1.0' encoding='UTF-8'?>
<BugCollection version='4.7.1' threshold='medium' effort='default'><file classname='org.waarp.ftp.client.DataTimeOutListener'><BugInstance type='EI_EXPOSE_REP2' priority='Normal' category='MALICIOUS_CODE' message='La méthode new org.waarp.ftp.client.DataTimeOutListener(FTPClient, long, String, String) risque d&apos;exposer sa représentation interne en stockant un objet externe modifiable dans org.waarp.ftp.client.DataTimeOutListener.client' lineNumber='52'/></file><file classname='org.waarp.ftp.client.WaarpFtp4jClient'><BugInstance type='REC_CATCH_EXCEPTION' priority='Normal' category='STYLE' message='La méthode org.waarp.ftp.client.WaarpFtp4jClient.connect() intercepte Exception, mais Exception n&apos;est pas lancé dans le bloc try/catch et RuntimeException n&apos;est pas appelé' lineNumber='251'/></file><file classname='org.waarp.ftp.client.WaarpFtpClient'><BugInstance type='DM_DEFAULT_ENCODING' priority='High' category='I18N' message='Found reliance on default encoding in new org.waarp.ftp.client.WaarpFtpClient(String, int, String, String, String, boolean, int, int, int, boolean): new java.io.PrintWriter(OutputStream)' lineNumber='132'/></file><Error></Error><Project><SrcDir>/home/frederic/Waarp2/Waarp-All/WaarpFtpClient/src/main/java</SrcDir><SrcDir>/home/frederic/Waarp2/Waarp-All/WaarpFtpClient/src/test/java</SrcDir></Project></BugCollection>